django/db/models/indexes.py

```diff
@@ -14,6 +14,9 @@ class Index(object):
     suffix = 'idx'
 
     def __init__(self, fields=[], name=None):
+        # Sanity checks for fields argument
+        if type(fields) != list:
```

Comment:
I'm not sure. What's the advantage of lists compared to tuples? I feel like most places in Django accept either although I'm not sure if there's a good reason for that other than backwards-compatibility.